### ISSUE TITLE
Network convergence when finetune on the other datasets

### ISSUE THREAD

**Issue by buqing2009** [mileyan/AnyNet#21]
Hi, thanks for sharing code. When I try finetune the AnyNet on sceneflow with [StereoDatasets](https://github.com/YuhuaXu/StereoDataset), it seems that the network can not converge to the small loss. I try to change lr and Loss function, but it cant help. Please help to analyse where is something wrong.
